config/configtls/configtls.go

```diff
@@ -373,13 +378,13 @@ func (c ClientConfig) LoadTLSConfig() (*tls.Config, error) {
 }
 
 // LoadTLSConfig loads the TLS configuration.
-func (c ServerConfig) LoadTLSConfig() (*tls.Config, error) {
+func (c *ServerConfig) LoadTLSConfig() (*tls.Config, error) {
```

Comment:
Goland yells at me when I mix receiver types. I believe best practice is not to mix types, so if this needs to be a pointer I think all the methods need updating.

Reply:
Done 👍